src/components/TrafficList/TrafficDetails.tsx

```diff
@@ -34,6 +34,7 @@ export interface WorkloadNode {
   type: NodeType.WORKLOAD;
   namespace: string;
   name: string;
+  healthAnnotation: any;
```

Comment:
Note that I put some comments in the backend that may affect to the UI code as well.
I'll try to add some early comments that I guess make sense skipping those related to naming described in

https://github.com/kiali/kiali/pull/3362

For this one, I guess this type can be a string instead of any.